coriolis/conductor/rpc/server.py

```diff
@@ -308,7 +308,8 @@ def _begin_tasks(self, ctxt, execution, task_info={}):
                     task_info=task_info.get(task.instance, {}))
 
     @replica_synchronized
-    def execute_replica_tasks(self, ctxt, replica_id, shutdown_instances):
+    def execute_replica_tasks(
+            self, ctxt, replica_id, shutdown_instances, force=False):
```

Comment:
As mentioned, "middle-tier" code should not make decisions on defaults for params which alter the overall logic/behavior.
Please define this param and its default in the API layer exactly as you did for the `force` flag for the update API call. (and everywhere else along the proverbial stack)
Currently, only Migrations-from-Replicas actually make use of that `force` param, but we might as well define it for all operations as future-proofing.